src/pages/graphile-build/hooks.md

```diff
@@ -20,54 +21,42 @@ const MyType = newWithHooks(GraphQLObjectType, spec);
 const MyType = new GraphQLObjectType(hook3(hook2(hook1(spec))));
 ```
 
-Hooks are registered via a call to `builder.hook(hookName, hookFunction)`.
+Hooks are registered via a call to [`builder.hook(hookName, hookFunction)`](/graphile-build/schema-builder/#hookhookname-hookfunction).
 
 Every `hookFunction` must synchronously return a value - either the value that it was passed
 or a derivative of it (preferably immutable, but we're not strict on that).
 
-### Which hook to attach to: `hookName`
+Hook functions for one hook by default run in the order they were registered, that is why the order of plugins is sometimes relevant.
```

Comment:
See https://github.com/graphile/graphile-engine/pull/416 - this might require an update. Probably the full description should reside at the `/graphile-build/schema-builder/#hookhookname-hookfunction`.